app/templates/canvas.template.html

```diff
@@ -1,3 +1,5 @@
 <div type="text/template" id="template-canvas">
     <canvas id="rectanglesCanvas" width="300" height="300" style="border: 1px solid black;"></canvas>
+
+    <button type="button" class="button" style="margin-left: 370px; margin-top: 300px" id="save">Save</button>

```

Comment:
what if i only need this specific button to move to a correct position? should i make additional class even if i need to change only margins?